packages/runtime-core/src/componentOptions.ts

```diff
@@ -368,7 +370,9 @@ export interface MethodOptions {
 export type ExtractComputedReturns<T extends any> = {
   [key in keyof T]: T[key] extends { get: (...args: any[]) => infer TReturn }
     ? TReturn
-    : T[key] extends (...args: any[]) => infer TReturn ? TReturn : never
+    : T[key] extends (...args: any[]) => infer TReturn
```

Comment:
Is this fixing something else? It's probably worth another test case

Reply:
Seems just re-formatted due to the upgrade of Prettier.

Reply:
oh right, I totally misread that